docs/UserGuide.adoc

```diff
@@ -486,63 +486,138 @@ You can then run `sort rate` to obtain the list of donors with higher success ra
+
+As introduced above, this command serves to help you mark a task on the task list as done.
+This will allow you to be able to know what other administrative things you need to do
+for the specific donor and patient before the cross-matching test.
+Beside each task on the task list, there will be either a cross, ✘, or a tick, ✓.
+A tick would mean that the task is completed and a cross will mean that the task is not completed.
+
+.Screenshot of the screen after you use the command on a valid donor and patient and a valid task number
+
+image::processingMarkDone.png[width="790"]
+
+Format: `processingMarkDone ic/**PATIENT NRIC** ic/**DONOR NRIC** **TASK NUMBER**` +
+The order of the NRICs does not matter, as long as the task number, patient and donor must be valid.
+The donor and patient need to be in `Processing` state to be considered valid.
+
+Example:
+`processingMarkDone ic/S6764235G ic/S5234567D 1`
+
+// end::process[]
+
+// tag::mark[]
+
 === Mark as done : `done`
 
 After the hospital has the results of the cross matching test, they can
-update the results in ORGANice. Cross matching tests have two possible results: pass or fail.
+update the results in ORGANice.
+Cross matching tests have two possible results: pass or fail.
 
-Format: `done icP/**NRIC OF PATIENT** icD/**NRIC OF DONOR** res/[pass/fail]`
-
-`icP/**NRIC OF PATIENT**` refers to the NRIC of the patient, while `icD/**NRIC OF DONOR**`
-refers to the NRIC of the donor.
+Format: `done ic/**PATIENT NRIC** ic/**DONOR NRIC** res/[pass/fail]`
+The order of the NRICs does not matter, as long as the patient and the donor must be valid.
 
 The following section outlines what happens based on the two results: +
 
 If the result of the cross matching is a 'pass':
 
-. It is up to the hospital staff to schedule an organ transplant surgery
+. Run the command based on the given format
+. It is up to the you to schedule an organ transplant surgery
 between the patient and donor.
-. Mark the patient and donor as 'done', based on the command format.
-. After running the command, the system removes the patient and donor from the matching pool.
-The patient and donor statuses will be marked to 'done'.
+. Mark the patient and donor as `Done`, based on the command format.
+. The patient and donor statuses will be marked to `Done`.
+. After running the command, the system removes the patient and donor from ORGANice.
 
 
 If the result of the cross matching is a 'fail': +
 
 . Run the command based on the given format
 
-. The patient and donor will be added back to the matching pool for
+. The donor and patient will be added back to the matching pool for
 match detection with other patients and donors.
 
-. The patient-donor pair will not be considered a potential match in future match detection.
+. The patient and donor statuses will be marked to `Not Processing`.
+
+. The donor and patient pair will not be considered a potential match in future match detection.
```

Comment:
suggested addition: potential match **anymore** in future match **detections**

Reply:
amended